build_linux.sh

```diff
@@ -9,7 +9,7 @@ export CODEC2DIR=$FREEDVGUIDIR/codec2
 export LPCNETDIR=$FREEDVGUIDIR/LPCNet
 
 # change this when working on combined codec2/freedv-gui changes
-CODEC2_BRANCH=master
+CODEC2_BRANCH=dr-rm-comp-varicode
```

Comment:
Ultimately this will end up in master, correct? Might not want to change this branch, or be sure to put it back once the other PR goes into master.

(BTW, "git add -p" will let you exclude certain changes from a pending commit.)

Reply:
Yes I will change it back before merge.  Doing it this way makes Travis use the correct codec2 PR when running it's tests.  There is probably a better way ....